### PR TITLE
adding kubectl GCE test to master blocking

### PR DESCRIPTION
**Pull request by jimangel**
Context from Slack (6/6/19) in #sig-release:
> liggitt [10:59 AM]
>qq about which test jobs are in release blocking... the kubectl skew test turned red about two weeks ago and no one noticed (https://testgrid.k8s.io/sig-cli-master#skew-cluster-latest-kubectl-stable1-gce&width=5) (edited) 
>turns out the test was flawed (fixing it now), but that is the sort of test I'd expect us to notice before releasing
>
> liggitt [11:06 AM]
> I just know that CI was called green in the 1.15 burndown this morning which made me suspect this job was not in release-blocking or release-informing
> 
> liggitt [11:06 AM]
> and I think it should be

Also:
> spiffxp [12:21 PM]
https://github.com/kubernetes/test-infra/pull/11245 @liggitt FWIW, SIG CLI was onboard with dropping the tests off, since they were on their own dashboard (not that I’m objecting to bringing them back on if they exercise useful functionality and will be owned)
>
>spiffxp [12:24 PM]
>yeah no objections to bringing it back, was just trying to add the history of how we got to now

Should the `alert_mail_to_addresses:` field be another team or will the release team suffice?

/priority critical-urgent
/milestone v1.15
/sig release
/kind cleanup
/area config
/cc @liggitt @spiffxp @alejandrox1

### CODE REVIEW

**Comment by liggitt** [comment]
Let's get the suite green first with https://github.com/kubernetes/kubernetes/pull/78772

**Comment by liggitt** [comment]
Oh, this is just adding to the dashboard. lgtm

File: testgrid/config.yaml [hunk 1]
```diff
@@ -2843,6 +2843,11 @@ dashboards:
     description: "Uses kubetest to run e2e tests (+Feature:Ingress|NEG) against a cluster created with cluster/kube-up.sh"
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
+  - name: skew-cluster-latest-kubectl-stable1-gce
+    test_group_name: ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew
+    description: stable1 e2e tests run against a master gce cluster using a stable1 kubectl binary
+    alert_options:
+      alert_mail_to_addresses: kubernetes-release-team@googlegroups.com
```
Comment (spiffxp) [inline testgrid/config.yaml]: is the release team really the right team to own this job? seems like this is sig-cli's responsibility

File: testgrid/config.yaml [hunk 2]
```diff
@@ -2843,6 +2843,11 @@ dashboards:
     description: "Uses kubetest to run e2e tests (+Feature:Ingress|NEG) against a cluster created with cluster/kube-up.sh"
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
+  - name: skew-cluster-latest-kubectl-stable1-gce
+    test_group_name: ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew
+    description: stable1 e2e tests run against a master gce cluster using a stable1 kubectl binary
```
Comment (spiffxp) [inline testgrid/config.yaml]: Can we be more specific about what this does?
```suggestion
    description: Uses kubetest to run e2e tests (+Kubectl, -Serial) from the previous release of kubernetes (stable1) against a cluster created with cluster/kube.sh from master
```

File: testgrid/config.yaml [hunk 3]
```diff
@@ -2843,6 +2843,11 @@ dashboards:
     description: "Uses kubetest to run e2e tests (+Feature:Ingress|NEG) against a cluster created with cluster/kube-up.sh"
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
+  - name: skew-cluster-latest-kubectl-stable1-gce
```
Comment (spiffxp) [inline testgrid/config.yaml]: Personally this name is hard to parse, what about some thing like?
```suggestion
  - name: gce-kubectl-skew
```

File: testgrid/config.yaml (hunk 1 above)
Comment (jimangel) [reply testgrid/config.yaml]: Updated to be sig-cli.

**Comment by jimangel** [comment]
> There were a number of other skew test jobs that were dropped as part of #11245, is this the only job we want to bring back?

It seems like we want the signal, but considering the bulk was ripped out, maybe it's good to only start with a single test? - Deferring to @liggitt or @alejandrox1

> If we're bringing this job into blocking for master, what about the other release-1.x boards?

I am for that, would it be a matter of updating the same file for those dashboards? Are there docs on how the release n+1 dashboard gets created after a release?

Sorry to answer your questions with more questions :)

**Comment by spiffxp** [comment]
AFAIK maintaining release dashboards once they've been forked with [`config-forker`](https://github.com/kubernetes/test-infra/blob/master/experiment/config-forker/README.md) is basically a manual process.  The argument being that whatever was used before for those releases is their problem, and we only focus on adjusting boards for the current release.  On the other hand, I'm a big fan of improving consistency and meaningful signal

/approve
/lgtm
Let's merge and iterate, but I don't think we're done here.

**Comment by spiffxp** [comment]
ping for an update, this job is still failing

**Comment by liggitt** [comment]
failing for different reasons now. the 1.15 job is testing deprecated APIs disabled in 1.16.

**Comment by liggitt** [comment]
related:
* https://github.com/kubernetes/kubernetes/pull/79747
* https://github.com/kubernetes/kubernetes/pull/79778
* https://github.com/kubernetes/kubernetes/issues/79740

**Comment by alejandrox1** [comment]
This one as well: 
* https://github.com/kubernetes/kubernetes/issues/79533 same failures as in sig-cli-master but in master-informing

**Comment by liggitt** [comment]
same cause... tests on an old branch calling things that don't make sense on a master branch API server

**Comment by liggitt** [comment]
looks like the upgrade parallel jobs are still running release-1.14 to master

**Comment by justaugustus** [comment]
@jimangel -- This job is green now. PR needs rebase and [RT added to alert annotations](https://github.com/kubernetes/test-infra/pull/12932#pullrequestreview-268140710).
